src/qutip_jax/convert.py

```diff
@@ -0,0 +1,19 @@
+import qutip
+from .jaxarray import JaxArray
+import jax
+import jax.numpy as jnp
+import numpy as np
+
+# Conversion function
+def jax_from_dense(dense):
+    return JaxArray(dense.to_array(), copy=False)
+
+def dense_from_jax(jax_array):
+    return qutip.data.Dense(np.array(jax_array), copy=False)
```

Comment:
A `JaxArray cannot be converted to a numpy array directly, we have a method for that.

```suggestion
    return qutip.data.Dense(jax_array.to_array(), copy=False)
```